### ISSUE TITLE
Init takes forever?

### ISSUE THREAD

**Issue by Derpford** [cloveranon/Clover-Edition#170]
I tried to run locally, ran the install script and it worked just fine, grabbed the model and installed it by extracting into the models folder, ran `play.bat` and...sat there. CPU usage was low, GPU usage was low, memory usage was kinda high, but it took a lot longer than a couple minutes.

**Comment by Skhmt** [comment]
Did it eventually work?

**Comment by Derpford** [comment]
I didn't let it run long enough to see if it worked. I'm running CPU only, with 16GB RAM (although thanks to lolwindows, that's more like 10GB at any given time).

I'm trying one of the GPT-Neo models, but I didn't have GPT-Neo enabled when I ran the install--should I try rerunning the installer?

**Comment by Skhmt** [comment]
No, if Neo didn't work, then it would crash. Plus, the installer only came around after Neo integration.

I'm thinking you're basically running out of RAM and it has to use swap space, which would make it take forever. Try running it again, but quit like literally everything you can on your PC.

**Comment by Derpford** [comment]
Weirdly enough, I closed everything and tried it again, and RAM usage capped out at around 5,505 MB on the process. I still had like 60% of my memory free. And CPU usage dropped drastically at around that point. Is there a toggle somewhere for limiting how much RAM the model can use?

**Comment by Skhmt** [comment]
We might have to increase the recommended RAM for CPU-only on GPT-Neo models.

**Comment by cloveranon** [comment]
possibly related to https://github.com/cloveranon/Clover-Edition/issues/158#issuecomment-836933632
